modules/core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/marshallers/json/document/DocumentPropertyJsonWriter.java

```diff
@@ -98,7 +98,7 @@
      */
     public static final String OMIT_PHANTOM_SECURED_PROPERTY = "omitPhantomSecuredProperty";
 
-    private static final Log log = LogFactory.getLog(DocumentPropertyJsonWriter.class);
+    protected static final Log log = LogFactory.getLog(DocumentPropertyJsonWriter.class);
```

Comment:
The logger should remain private, otherwise, the class extending this one could use it to log with inappropriate fqdn.

Reply:
ok. I reverted it to private.